manifests/get.pp

```diff
@@ -115,6 +120,14 @@
     $myChown = undef
   }
 
+  if $chmod {
+    $myChmod = "--chmod=${chmod}"
```

Comment:
Whoops, sorry about that.  Done.

I think my Puppet Lint updated since I last committed changes to get.pp - it's complaining about variables containing uppercase letters.  I updated the ones I added not to contain them, but there are 35 other errors.  Is it okay to go through and update that as part of this PR, or would you prefer for me to disable lint temporarily?